spec/system/budgets/investments_spec.rb

```diff
@@ -661,6 +661,16 @@ def investments_order
           expect(page).not_to have_content "You are seeing"
         end
       end
+
+      scenario "Don't show suggestions for edit action", :js do
+        investment = create(factory, title: "Title, has search term", budget: budget, author: author)
+
+        login_as(author)
+        visit edit_budget_investment_path(budget, investment)
+        fill_in "Title", with: "search", fill_options: { clear: :backspace }
```

Comment:
`clear: :backspace` won't be needed after pull request #4456. Shall we rebase this pull request so it depends on #4456?